src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -54,6 +55,9 @@ public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossT
           tenantsToQuery.add(tenantId);
         } catch (MissingPermissionsException e) {
           log.info("User with id {} does not have permissions to query tenant {}. Skipping.", userId, tenantId);
+        } catch (FeignException e) {
+          log.error("Error retrieving permissions for user ID %s in tenant %s".formatted(userId, tenantId), e);
```

Comment:
Log this, to provide a little more useful info that isn't in the exception that shows up right now (then re-throw it, since we don't want to fail silently here)